docs/CONFIG-PROPERTIES.md

```diff
@@ -50,6 +50,9 @@
 | memory.vmm.limit.MiB | integer | 0 | Manually override how much overhead is allocated for each running VMM |
 | gogc.memory.limit.bytes | integer | 0 | Golang runtime soft memory limit, see details in API doc ["https://pkg.go.dev/runtime/debug#SetMemoryLimit"] |
 | gogc.percent | integer | 100 | Golang runtime garbage collector target percentage, see details in API doc ["https://pkg.go.dev/runtime/debug#SetGCPercent"] |
+| gogc.forced.period.seconds | integer in seconds | 10 | period of forced execution of the GC. Forced GC is disabled when 0 is set |
```

Comment:
Sorry for the naming nits, but I find it odd to refer to this as a period when the description explains it as being the minimum time between forced GC calls. Hence there is no periodic activity AFIACT.

Reply:
Period wording makes sense if we constantly receive those pressure events. Interval also fine to me. Or minimum time. Just don't want to increase length of the parameter string.